src/main/java/pl/byczazagroda/trackexpensesappbackend/model/User.java

```diff
@@ -56,11 +56,11 @@ public class User implements Serializable {
 
     @NotBlank
     @Size(min = 6, max = 120)
-    @Pattern(regexp = "^[\\w-.]+@([\\w-]+.)+[\\w-]{2,4}$")
+    @Value("${regex.pattern.email}")
```

Comment:
Check if this works as we expected.

Reply:
From the spring documentation for the Value annotation:

Note that actual processing of the @Value annotation is performed by a [BeanPostProcessor](https://docs.spring.io/spring-framework/docs/current/javadoc-api/org/springframework/beans/factory/config/BeanPostProcessor.html) which in turn means that you cannot use @Value within [BeanPostProcessor](https://docs.spring.io/spring-framework/docs/current/javadoc-api/org/springframework/beans/factory/config/BeanPostProcessor.html) or [BeanFactoryPostProcessor](https://docs.spring.io/spring-framework/docs/current/javadoc-api/org/springframework/beans/factory/config/BeanFactoryPostProcessor.html) types. Please consult the javadoc for the [AutowiredAnnotationBeanPostProcessor](https://docs.spring.io/spring-framework/docs/current/javadoc-api/org/springframework/beans/factory/annotation/AutowiredAnnotationBeanPostProcessor.html) class (which, by default, checks for the presence of this annotation).

This means that the Value annotation applies to the bean, not the entity. Additionally, the Value annotation has a different purpose than the Pattern annotation.